### PR TITLE
MGMT-2628: Misleading message to retry installation on specific host

### PR DESCRIPTION
**Pull request by jkilzi**
- Omits the "Debug and reboot each host to try again." message when the cluster has finished and has a failing host.

### CODE REVIEW

**Review by mareklibra — DISMISSED** [review]

File: src/components/clusterDetail/FailedHostsWarning.tsx [hunk 1]
```diff
@@ -47,6 +47,11 @@ const FailedHostsWarning: React.FC<FailedHostsWarningProps> = ({ cluster }) => {
       ' The cluster installation will time-out after 30 minutes if a master host does not reboot properly.';
   }
 
+  let failedHostsText = 'Error information for each worker host is available below.';
+  if (cluster.status !== 'finalizing') {
+    failedHostsText += ' Debug and reboot each host to try again.';
```
Comment (jtomasek) [inline src/components/clusterDetail/FailedHostsWarning.tsx:52]:
```suggestion
    failedHostsText += ' Debug and reboot each failed worker to retry their installation.';
```

File: src/components/clusterDetail/FailedHostsWarning.tsx [hunk 2]
```diff
@@ -47,6 +47,11 @@ const FailedHostsWarning: React.FC<FailedHostsWarningProps> = ({ cluster }) => {
       ' The cluster installation will time-out after 30 minutes if a master host does not reboot properly.';
   }
 
+  let failedHostsText = 'Error information for each worker host is available below.';
```
Comment (jtomasek) [inline src/components/clusterDetail/FailedHostsWarning.tsx:50]:
```suggestion
  let failedHostsText = 'Error information for each host is available below.';
```

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: src/components/clusterDetail/FailedHostsWarning.tsx (hunk 1 above)
Comment (jkilzi) [reply src/components/clusterDetail/FailedHostsWarning.tsx]: resolved in 699c3cd7586fe1a80e1ea8ec0b2cbbd8024c9f34

File: src/components/clusterDetail/FailedHostsWarning.tsx (hunk 2 above)
Comment (jkilzi) [reply src/components/clusterDetail/FailedHostsWarning.tsx]: resolved in 699c3cd7586fe1a80e1ea8ec0b2cbbd8024c9f34

**Review by mareklibra — APPROVED** [review]

**Review by mareklibra — APPROVED** [review]

**Pull request merged**